Workbooks/Communication Services/Overview/Overview.workbook

```diff
@@ -0,0 +1,1030 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Communication Service"
+      },
+      "name": "general-title"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "b1841bbb-0dec-444a-9873-ee90e6749e99",
+            "version": "KqlParameterItem/1.0",
+            "name": "time_range",
+            "label": "Time range",
+            "type": 4,
+            "description": "Define a time range to aggregate logs over",
+            "isRequired": true,
+            "isGlobal": true,
```

Comment:
does this have to be Global? If time range is not reassigned somewhere further down the workbook, then this can just be a normal parameter with the value that flows downstream

https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Parameters/Parameters.md#global-parameters

Reply:
Changed to non-global parameters. Thanks for the clarification!